src/d3d9/d3d9_device.cpp

```diff
@@ -4755,12 +4755,12 @@ namespace dxvk {
       UploadSoftwareConstantSet(slice.mapPtr, Src, Layout, Shader);
 
     if (constSet.meta.needsConstantCopies) {
-      Vector4* data = reinterpret_cast<Vector4*>(slice.mapPtr);
+      float* data = reinterpret_cast<float*>(slice.mapPtr);
```

Comment:
The mapped pointer will be page aligned to 4096 so this should be okay to stay as Vector4?

Reply:
Same for other uploads to a buffer ig.

Reply:
Removed the commit